packages/Core/Panic.candy

```diff
@@ -1 +1 @@
-panic message = ✨.panic message
+panic := { message -> ✨.panic message }
```

Comment:
Good question. I thought this would have made some sense, but I can't remember why.
Another idea: What do you think about implementing panic like this?

```candy
panic message :=
  needs (isText message) "panic needs a text as the message"
  needs False message
```